OpenCover.UI.TestDiscoverer/NUnit/NUnitDiscoverer.cs

```diff
@@ -111,7 +135,11 @@ private TestMethod[] DiscoverTestsInClass(TypeDefinition type, TestClass @class)
 			foreach (var method in type.Methods)
 			{
 				bool isTestMethod = false;
-				var trait = new List<string>();
+                var traits = new List<string>();
+                if (@class.Traits != null && @class.Traits.Length > 0)
+                {
+                    traits.AddRange(@class.Traits);
+                }
```

Comment:
Thanks Peter, that will get me going.  On the projects that use NUnit, we have been installing the test adapter through Nuget packages so there was no need to add it directly to Visual Studio.  We also are using NUnit 3, and it appears you are using NUnit 2, so I'm not sure how these adapters are going to play together.  Oh well, sigh.